charts/apisix/values.yaml

```diff
@@ -305,6 +305,24 @@ apisix:
     # ref: https://apisix.apache.org/docs/apisix/deployment-modes/
     role: "traditional"
 
+    # -- Standalone rules configuration
+    #
+    # ref: https://apisix.apache.org/docs/apisix/deployment-modes/#standalone
+    standalone:
+      # -- Rules which are set to the default apisix.yaml configmap.
+      # -- if apisix.delpoyment.standalone.existingConfigMap is empty, these are used.
+      config: |
+        routes:
+        -
+          uri: /hi
+          upstream:
+            nodes:
+              "127.0.0.1:1980": 1
+            type: roundrobin
+      # -- Specifies the name of the ConfigMap that contains the rule configurations.
+      # The configuration must be set to the key named `apisix.yaml` in the configmap.
+      existingConfigMap: ""
```

Comment:
Is this really needed?

Reply:
It is for this subchart issue: https://github.com/apache/apisix-helm-chart/issues/705

Though I have not tested if this is really needed for that use case. But I think there are many reasons that some one would like to use a configmap that is created outside of this chart. For example related to devops automation, where the configuration is defined/updated separate to the chart installation.

Reply:
We need this as well. The current Helm chart adds a apisix.yaml ConfigMap when it is installed in standalone mode. This ConfigMap contains a /hi route that is impossible to modify or remove. We need the ability to specify our own rule config instead of this example config.

One workaround would maybe be to "overwrite" the example config mount using `extraVolumes` and `extraVolumeMounts` but I much rather prefer this solution.